### PR TITLE
Presentation/non standard URL identification

### PR DESCRIPTION
**Pull request by aurislt7**

### CODE REVIEW

**Review by grigasp — APPROVED** [review]

File: common/changes/@bentley/ui-components/presentation-URL-values_2021-02-02-14-22.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/ui-components",
+      "comment": "Added a hack for identifying non-standard URL values",
```
Comment (grigasp) [inline common/changes/@bentley/ui-components/presentation-URL-values_2021-02-02-14-22.json:5]:
```suggestion
      "comment": "Consider string values starting with `pw:\\` or `pw://` to be URLs without checking the rest of the string.",
```

**Review by DanEastBentley — APPROVED** [review]

**Comment by grigasp** [comment]
@mergifyio backport release/2.12.x

**Pull request merged**